src/main/kotlin/ru/kode/detekt/rule/compose/UnnecessaryLayoutWrapper.kt

```diff
@@ -0,0 +1,75 @@
+package ru.kode.detekt.rule.compose
+
+import io.gitlab.arturbosch.detekt.api.CodeSmell
+import io.gitlab.arturbosch.detekt.api.Config
+import io.gitlab.arturbosch.detekt.api.Debt
+import io.gitlab.arturbosch.detekt.api.Entity
+import io.gitlab.arturbosch.detekt.api.Issue
+import io.gitlab.arturbosch.detekt.api.Rule
+import io.gitlab.arturbosch.detekt.api.Severity
+import org.jetbrains.kotlin.psi.KtBlockExpression
+import org.jetbrains.kotlin.psi.KtCallExpression
+import org.jetbrains.kotlin.psi.KtFile
+import org.jetbrains.kotlin.psi.KtLambdaArgument
+import org.jetbrains.kotlin.psi.KtNameReferenceExpression
+import org.jetbrains.kotlin.psi.psiUtil.getChildOfType
+import org.jetbrains.kotlin.psi.psiUtil.getChildrenOfType
+
+class UnnecessaryLayoutWrapper(config: Config = Config.empty) : Rule(config) {
```

Comment:
Please add examples of compliant and non compliant code (we'll use them later to generate docs).

See example [here](https://github.com/appKODE/detekt-rules-compose/blob/master/src/main/kotlin/ru/kode/detekt/rule/compose/ComposableFunctionName.kt#L21)